code/combine_duplicate.py

```diff
@@ -0,0 +1,70 @@
+import sys
+
+#User enters the samples that it wants to use (Ex. 01A, 10A, 11A)
+file_ids = sys.argv[1]
+
+
+#Path to file which has repeating rsIDs that need to be combined
+file_path = "/nobackup/shilab/Data/TCGA/LUAD/analysis/tcga-luad-profiles/code/rs_id_" + file_ids + ".txt"
```

Comment:
This should be changed so we don't have any hardcoded paths in the code. 
The user could just be the complete path in `file_ids`